### DISCUSSION TITLE
blog/meshtastic-opposition-to-nextnav-proposed-changes/

### DISCUSSION THREAD

**Discussion by giscus** [meshtastic/Discussions#16] [Blog Post Comments]
# blog/meshtastic-opposition-to-nextnav-proposed-changes/

Why We Oppose NextNav's 900 MHz Proposal: A Threat to Open Communication Networks

https://meshtastic.org/blog/meshtastic-opposition-to-nextnav-proposed-changes/

**Comment by captainchaosrules** [comment]
OMG what are the odds that the FCC approves this Next Nav request?

**Comment by blarson** [comment]
You can read NextNav’s petition here: https://www.fcc.gov/ecfs/document/10416238018537/1

You can file an express comment in opposition to the proposal here: https://www.fcc.gov/ecfs/filings/express?proceeding%5Bname%5D=24-240

**Reply by captainchaosrules** [reply]
Would it be better to file as a small business or as an individual or perhaps both?

**Reply by blarson** [reply]
I suppose filing as a small business would carry a bit more weight, but you should probably submit a single filing and mention both your business interest and personal interest in preserving the current rules.

**Comment by captainchaosrules** [comment]
The power levels of NextNav and 5G systems would de-sense the Meshtastic (LoRa) Radios degrading the range substantially.

**Reply by blarson** [reply]
That's my understanding as well. The only mention of power level in their petition I can find is something about 30 W effective radiated power for ground stations (P.12). If that's what they're going to be doing with ground stations, and they'll presumably be blanketing cities and possibly rural areas with constellations of ground stations to provide the precision locating as advertised (and also the "mobile broadband" they also want to deploy), that's going to absolutely hammer the band and shout down or jam anyone using 20mW, 50mW, or 100mW LoRa devices.

This seems like a backdoor attempt to carve out some commercial broadband for themselves while pretending they can play nice with secondary users. They want to buy up all the M-LMS licenses, then convert them to a national license, and then go beyond the bounds and the rules of M-LMS to do something entirely different than what the band was allocated for.

Their comments about how the changes will affect other band users are laughable, to be honest.

_"In 2013, the American Radio Relay League acknowledged “M-LMS ‘was already allocated with a higher priority’” and opted not to file comments in the proceeding",_ therefore _"Updating the Lower 900 MHz Band M-LMS framework will not impede amateur operations."_

Nonsensical. The ARRL was commenting on the current status of the band rules in 2013, not NextNav's proposal to overhaul the band rules in 2024.

_"Coexistence between the NextGen system and unlicensed Part 15 operations should be achievable."_ But then _"NextNav is completing technical analyses intended to address representative Lower 900 MHz Part 15 uses"_. So they conclude that there shouldn't be a problem with stepping on Part 15 users, but they haven't done the analysis yet. They're still working on it. But trust them, there shouldn't be a problem.

_"Part 15 devices must coexist with one another in addition to other operations in the band. As a result, Part 15 devices need to be robust and adapt to ever-present interference, including mutual Part 15 interference, which is unpredictable by nature."_

Effectively, what they're saying here is that Part 15 devices are required to tolerate interference from devices operating within the current band rules, therefore they have no expectation of successful operation under the new band rules proposed for NextNav's new use cases.

So, there shouldn't be a problem, but even if there is, tough luck, Part 15 users.

**Reply by captainchaosrules** [reply]
https://g.co/gemini/share/c49980406b82

**Comment by captainchaosrules** [comment]
https://www.arrl.org/news/arrl-urges-protecting-the-amateur-radio-902-928-mhz-band

**Comment by Steve-AG2AA** [comment]
Has meshtastic considered coverting the letter of opposition to a petition, like on change.org or some other platform so it would be easier to engage the mesh users on a large scale? a large number of opponents would be the only way to effectively oppose the deep pockets of a commercial applicant like nextnav.

**Comment by Planedude** [comment]
Homeland Security Admin has stated in the past that LORA, specifically Meshtastic, is a security threat due to encryption on the band. Don't think this is a coincidence. Perhaps, this is an attempt to thwart Meshtastic private encryption and basically shut it down. So, this an easy way, in the name of "public safety" to shut down Meshtastic. Or maybe, require individuals to purchase a license to help fund the overlords of the airways. :-)

**Comment by shawnyeager** [comment]
Thank you for bringing this to our attention. Comment submitted to the FCC. I encourage others to do the same here:

https://www.fcc.gov/ecfs/filings/express?proceeding%5Bname%5D=24-240&mc_cid=51139ac370

**Comment by yaeloss** [comment]
My organization, the Consumer Choice Center, filed its comments with the FCC to oppose this proposal and to support amateur radio and decentralized LoRa projects:

https://consumerchoicecenter.org/consumer-choice-center-submits-fcc-comments-to-protect-radio-frequency-for-open-source-projects-and-amateur-radio/

**Comment by OogieM** [comment]
For me the opposition letter downloads as a corrupted file. Is there another place to see it?

**Reply by yaeloss** [reply]
do you mean on the FCC portal website?

**Comment by SimBacon** [comment]
### ! TODAY ! Sept 20 2024 

is the last day to file a comment! Pile on and let them know that NextNav should not be allowed to band-grab 902-928 MHz!

Submit your reply here: https://www.fcc.gov/ecfs/filings/express?proceeding%5Bname%5D=24-240&mc_cid=51139ac370

**Comment by 1newRodldavis** [comment]
FCC document DA 24-776 is a request for information from Aug 2024. I do't know if it is still open, if it is flood them with comments. 
NextNav has already purchased frequencies that they can use why infringement on what little we have to use
